### PR TITLE
SYS-1642 Move avn-proxy to this repo

### PR DESCRIPTION
**Pull request by nahuseyoum**
This PR moves avn-proxy to this repo. There are no functionality changes.

### CODE REVIEW

**Review by fluorostani — APPROVED** [review]

File: pallets/avn-proxy/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,226 @@
+//! # Avn proxy pallet
+// Copyright 2020 Aventus Network Services (UK) Ltd.
```
Comment (thadouk) [inline pallets/avn-proxy/src/lib.rs:2]:
```suggestion
// Copyright 2022 Aventus Network Services (UK) Ltd.
```
Probably worth updating the copyright here

File: pallets/avn-proxy/src/benchmarking.rs [hunk 2]
```diff
@@ -0,0 +1,84 @@
+//! # avn-proxy
+// Copyright 2021 Aventus Network Systems (UK) Ltd.
```
Comment (thadouk) [inline pallets/avn-proxy/src/benchmarking.rs:2]:
```suggestion
// Copyright 2022 Aventus Network Systems (UK) Ltd.
```

File: pallets/avn-proxy/src/tests/mock.rs [hunk 3]
```diff
@@ -0,0 +1,369 @@
+//Copyright 2021 Aventus Network Services (UK) Ltd.
```
Comment (thadouk) [inline pallets/avn-proxy/src/tests/mock.rs:1]:
```suggestion
//Copyright 2022 Aventus Network Services (UK) Ltd.
```

File: pallets/avn-proxy/src/tests/proxy_tests_no_fees.rs [hunk 4]
```diff
@@ -0,0 +1,157 @@
+//Copyright 2021 Aventus Network Systems (UK) Ltd.
```
Comment (thadouk) [inline pallets/avn-proxy/src/tests/proxy_tests_no_fees.rs:1]:
```suggestion
//Copyright 2022 Aventus Network Systems (UK) Ltd.
```

File: pallets/avn-proxy/src/tests/proxy_tests_with_fees.rs [hunk 5]
```diff
@@ -0,0 +1,499 @@
+//Copyright 2021 Aventus Network Systems (UK) Ltd.
```
Comment (thadouk) [inline pallets/avn-proxy/src/tests/proxy_tests_with_fees.rs:1]:
```suggestion
//Copyright 2022 Aventus Network Systems (UK) Ltd.
```

**Review by thadouk — APPROVED** [review]
I think Cargo.lock will also need updating.

**Pull request merged**